### PR TITLE
feat: validation by fluent validation

### LINKED ISSUE #60: Валидация входных данных
Проверка длинны строк, типов файлов (если надо обложку на трек - это должна быть картинка, если трэк - то звуковой формат) и прочее

### PR DESCRIPTION
**Pull request by JabaJabila**
## Type
- [x] new
- [ ] fix
- [ ] refactor

## Interface
- [ ] frontend
- [x] backend
- [ ] docs
- [ ] ci

closes #60

### CODE REVIEW

File: Validators/Playlist/CreateNewPlaylistCommandValidator.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+namespace Validators.Playlist;
+
+public class CreateNewPlaylistCommandValidator : AbstractValidator<CreateNewPlaylist.CreateNewPlaylistCommand>
+{
+    private static readonly string[] PictureFormats = { ".jpg", ".png", ".jpeg", ".heic" };
+    private const int MinNameLength = 5;
+    private const int MaxNameLength = 20;
+    private const int MaxDescriptionLength = 500;
+    
+
```
Comment (sashafromlibertalia) [inline Validators/Playlist/CreateNewPlaylistCommandValidator.cs:14]: отступ лишний

**Review by sashafromlibertalia — CHANGES_REQUESTED** [review]
супердоебался

File: Validators/Playlist/CreateNewPlaylistCommandValidator.cs (hunk 1 above)
Comment (JabaJabila) [reply Validators/Playlist/CreateNewPlaylistCommandValidator.cs:14]: (

**Review by Ssspook — APPROVED** [review]
И есчо раз

**Review by sashafromlibertalia — APPROVED** [review]

**Pull request merged**